manager/src/main/resources/db/migration/0.1/V0.1.3__create_connectors.sql

```diff
@@ -1,13 +1,13 @@
 create table CONNECTOR
 (
-    id                 varchar(255) NOT NULL PRIMARY KEY,
-    processor_id       varchar(255) NOT NULL,
-    name               varchar(255) NOT NULL,
-    submitted_at       timestamp    NOT NULL,
-    published_at       timestamp,
-    status             varchar(255),
-    version            integer      NOT NULL default 0,
-    definition         jsonb        NOT NULL,
+    id                          varchar(255) NOT NULL PRIMARY KEY,
```

Comment:
@lucamolteni You'll need to fully revert this file so there are no perceived changes. I think Flyway will fail if there is additional whitespace IIRC as it will alter the `checksum` of the migration and will not be the same as the one already applied.